settings.py

```diff
@@ -6,11 +6,33 @@
 values from localbuildsettings.py (if exists).
 """
 
+import time
+
+
+def get_current_timestamp():
+    """Get current build date and timestamp in desired formats."""
+    utc = time.gmtime()
+    return (
+        time.strftime('%Y-%m-%d-%H%M%S', utc),  # build_date
+        time.strftime('%Y%m%d.%H%M%S', utc)     # build_timestamp
+    )
+
+
+def build_timestamp():
```

Comment:
Would it be better or more readable to call these functions from the get_current_timestamp method and have the actual building here?

Reply:
Do you propose to get rid of `get_current_timestamp()`, move the code to `build_timestamp()` and `build_date()`?
Maybe. I like both options, there are pros and cons to both

Reply:
Both have a race condition.  If the second ticks over between the calls to `gmtime()`, the results will be different.

I don't know if there is any code that depends on those values being identical.

Reply:
> Do you propose to get rid of `get_current_timestamp()`, move the code to `build_timestamp()` and `build_date()`?
> Maybe. I like both options, there are pros and cons to both

Well, no... do not remove the get_current_timestamp.
Rather to have the `time.strftime(...)` inside of the "build" methods which would get called from get_current_timestamp with the same "time" parameter and have this parameter default to time.utc in case its called separately.

The point is to not calculate both when we need only one of those.

(This is just about polishing the code, I'm quite ok with the current state, it just feels weird to call method to calculate several values and then use only one 🤷‍♂️)

Reply:
No need to block the PR for this, but if it could be improved I'd be glad 😉